packages/eslint-plugin/docs/rules/restrict-plus-operands.md

```diff
@@ -16,12 +16,34 @@ var foo = 1n + 1;
 
 ## Options
 
+Options may be provided as an object with:
+
+- `allowAny` to ignore this rule when either left or
+  right of plus operand is a type `any`
+
 ```json
 {
-  "@typescript-eslint/restrict-plus-operands": "error"
+  "@typescript-eslint/estrict-plus-operands": [
+    "error",
+    {
+      "allowAny": true
+    }
+  ]
 }
 ```
 
+### allowAny
+
+If the rule is too strict then making this option `true`
+can be a help. Though It is not recommended since lint errors are potentially a real runtime errors in many cases.
```

Comment:
Suggestion for stronger/clearer wording for this:

----

***This option is not recommended, as it greatly reduces the type safety in your code.***
***It is provided as a stopgap to make it easier to migrate codebases onto the rule.***

This option will cause the rule to accept `any` as a type for operands. `any` will be allowed to be summed with all other valid types.